src/main.rs

```diff
@@ -117,7 +127,7 @@ fn main() {
         if let Some(duration) = opt.for_duration {
             let since = Instant::now().duration_since(program_start);
             if since >= duration {
-                return;
+                break;
```

Comment:
Why are these change?

Reply:
😄 it stomped me too for a few minutes...